platform/ios/src/MGLMapView.h

```diff
@@ -1431,6 +1433,16 @@ MGL_EXPORT IB_DESIGNABLE
  */
 - (void)selectAnnotation:(id <MGLAnnotation>)annotation animated:(BOOL)animated;
 
+/**
+ :nodoc:
```

Comment:
What's the intention behind including this method in the public header but `nodoc`’ing it? If this is a method we intend for developers to use, what do we need to do to before making it fully public/documented?